src/lib/openapi/spec/archived-feature-schema.ts

```diff
@@ -0,0 +1,103 @@
+import type { FromSchema } from 'json-schema-to-ts';
+
+export const archivedFeatureSchema = {
+    $id: '#/components/schemas/archivedFeatureSchema',
+    type: 'object',
+    additionalProperties: true,
```

Comment:
Why have we set this to true? Do we expect to add additional properties? The default for data that we return is to set it to `false` so that we don't accidentally return more than what we say we will.

Reply:
i was testing sth else and forgot to bring it back to false. Thanks for catching